config/initializers/f_foreman_permissions.rb

```diff
@@ -562,8 +562,8 @@
       :users => [:new, :create],
       :"api/v2/users" => [:create]
     map.permission :edit_users,
-      :users => [:edit, :update],
-      :"api/v2/users" => [:update]
+      :users => [:edit, :update, :invalidate_jwt],
+      :"api/v2/users" => [:update, :invalidate_jwt]
```

Comment:
Where does `invalidate_jwt` come from? I don't see it defined in `PermissionsList`

Reply:
it's an action in userscontrollers, do we need to add it in the permissions list?